### PR TITLE
Issue 1 remove custom scroll

### LINKED ISSUE #6: Remove custom scroll bar from screens
Remove custom scroll bar from screens
add responsiveness to products page

### PR DESCRIPTION
**Pull request by aishwarya-bm**
1.Removed custom scroll bars
2.added style for brand name
3.remove the alternate page for products(horizontal)

### CODE REVIEW

**Review by dev-enforced — APPROVED** [review]
The code looks good to me !!
And the title of Art Station also looks good!!
Good work @aishwarya-bm on the authentication component!!

**Comment by aishwarya-bm** [comment]
> The code looks good to me !! And the title of Art Station also looks good!! Good work @aishwarya-bm on the authentication component!!

Thank you, Roshan! :)

**Comment by aishwarya-bm** [comment]
In the latest commit #[f7f36b8](https://github.com/aishwarya-bm/e-commerce/pull/7/commits/f7f36b8c5996ed573bfc9cb095b8c31aef38cf8e) I have removed the alternate page for products(horizontal)

File: style.css [hunk 1]
```diff
@@ -1,9 +1,50 @@
 @import url(./stylesheets/main.css);
 
+.nav-title{
+        font-size: 2rem;
+        font-family: cursive;
+        /* letter-spacing: -7px; */
```
Comment (Sameony) [inline style.css]: do remove comments and align code before merging

File: stylesheets/screens/productslist.css [hunk 2]
```diff
@@ -115,6 +100,24 @@
...
+    }
+    .nav-container .btn-modal-alert{
+        display: block;
+    }
+    .category-name{
+        text-align: center;
+    }
+    /* .filter-container{
+        display: none;
+    } */
```
Comment (Sameony) [inline stylesheets/screens/productslist.css]: this block too

**Review by Sameony — APPROVED** [review]
great code. do follow up on the suggestions and merge it

File: stylesheets/screens/productslist.css [hunk 3]
```diff
@@ -82,25 +73,19 @@
...
 }
-  
-@media only screen and (max-width:550px) {
-    .productlist-body {
-        overflow-y: auto;
-    }
-    
+
+.nav-container .btn-modal-alert {
+    display: none;
```
Comment (dev-enforced) [inline stylesheets/screens/productslist.css:89]: can use this as a common utility class in your styles.css file and passing it in the HTML file

**Review by dev-enforced — APPROVED** [review]
The responsive modal for filtering products looks good and the appropriate code used to develop it is also good

File: style.css (hunk 1 above)
Comment (aishwarya-bm) [reply style.css]: Thank you! Made the changes!

File: stylesheets/screens/productslist.css (hunk 3 above)
Comment (aishwarya-bm) [reply stylesheets/screens/productslist.css:89]: util class for display requires JS but I dont have time for JS now, so handled using CSS

**Pull request merged**